aggregator/persist.go

```diff
@@ -45,6 +45,8 @@ func (a *Aggregator) processPod(d k8s.K8sResourceMessage) {
 		OwnerType: ownerType,
 		OwnerID:   ownerID,
 		OwnerName: ownerName,
+
+		Labels: pod.GetLabels(),
```

Comment:
Here we keep track of Pod labels in our local cache of k8s resources. This is useful for putting Acorn-specific information onto the metrics, like app, container, and project names.